docs/posts/ibis-version-9.0.0-release/index.qmd

```diff
@@ -0,0 +1,486 @@
+---
+title: "Ibis 9.0: SQLGlot-ification"
+author: "Ibis team"
+date: "2024-05-01"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis 9.0 wraps up ["the big
+refactor"](../../posts/roadmap-2024-H1/index.qmd#the-big-refactor), completing
+the transition from SQLAlchemy to [SQLGlot](https://github.com/tobymao/sqlglot)
+and drastically simplifying the codebase. This is a big step toward stabilized
+internals and allows us to more easily add new features and backends going
+forward. Look out for a dedicated blog post on the refactor soon.
+
+:::{.callout-tip}
+One long-standing issue with Ibis has been chained joins, which now work as
+a user would expect. This was a major motivation for the refactor.
+:::
+
+Ibis 9.0 also adds new functionality, fixes many bugs, and welcomes two new
+committers to the project!
+
+### Meet the Ibis team!
+
+Members of the core Ibis team will be at [PyCon 2024 in Pittsburg, PA,
+USA in May](https://us.pycon.org/2024/) and [SciPy 2024 in Tacoma, WA,
+USA in July](https://www.scipy2024.scipy.org/)! Stop by at one of the talks or
+tutorials to say hi in English and/or Español:
+
+- [Tutorials: Introduction to Ibis: blazing fast analytics with DuckDB, Polars,
+ Snowflake, and more, from the comfort of your Python
+ repl.](https://us.pycon.org/2024/schedule/presentation/55/)
+- [Charlas: Porque SQL está en todas
+ partes.](https://us.pycon.org/2024/schedule/presentation/117/)
+
+::: {.callout-note}
+The schedule for SciPy 2024 is not available yet, keep an eye out for Ibis sessions.
+:::
+
+You can also listen to a [recent podcast episode with Real
+Python](https://realpython.com/podcasts/rpp/201) where Phillip Cloud, lead
+maintainer of Ibis, talks about the project.
+
+More podcasts and talks are in the works, so stay tuned!
+
+### New committers
+
+We're excited to welcome [Nick Crews](https://github.com/NickCrews) and [Naty
+Clementi](https://github.com/ncclementi) as the newest committers to the Ibis
```

Comment:
Maybe make `committers` a link to our governance doc, to indicate that here "committers" means a level of privileges rather than just "has made commits"?